### ISSUE TITLE
Provide sidecar for k8 usage (and document it)

### ISSUE THREAD

**Issue by WtfJoke** [codecentric/chaos-monkey-spring-boot#288]
I made tests using kubernetes and chaos monkey can used that way. In order to use that we should publish a sidecar container for public use and publish it in docker hub/github container registry

See example
https://github.com/WtfJoke/chaoskotlindemo/commit/2fef52a904faed9e78f794eb873e4a0b23aed645

**Comment by checketts** [comment]
Sidecars are useful when you can't include it as a library. Is that the goal? To use chaosmonkey features with node or go apps?

**Comment by WtfJoke** [comment]
> Sidecars are useful when you can't include it as a library. Is that the goal?

Yes, the current extjar would be provided as as a docker image which can be used as a sidecar for an existing spring application.


> To use chaosmonkey features with node or go apps?

I dont know what you mean with this (we can only assault spring boot apps with jvm languages) 😅

**Comment by checketts** [comment]
Oh I see. The sidecar isn't a process that is used, just a mechanism to add files (the jar)

**Comment by checketts** [comment]
> >   To use chaosmonkey features with node or go apps?

> I dont know what you mean with this (we can only assault spring boot apps with jvm languages) 😅

Which is why I was confused. Thanks for clarifying it for me.
